src/improvements/tasks/sep/007-opcm-upgrade-v300-uni/VALIDATION.md

```diff
@@ -0,0 +1,203 @@
+```
+
+In mainnet runbooks, this calldata should appear in [Action Plan](https://gov.optimism.io/t/upgrade-proposal-14-isthmus-l1-contracts-mt-cannon/9796#p-43948-action-plan-9) section of the Governance proposal.
+
+# State Changes
+
+## Single Safe State Overrides and Changes
+
+This task is executed by the single `ProxyAdminOwner` Safe. Refer to the [generic single Safe execution validation document](../../../../../SINGLE-VALIDATION.md)
+for the expected state overrides and changes.
+
+Additionally, Safe-related nonces [will increment by one](../../../../../SINGLE-VALIDATION.md#nonce-increments).
+
+## Other Nonces
+
+In addition to the Safe-related nonces mentioned [previously](#single-safe-state-overrides-and-changes), new contracts will also have a nonce value increment from 0 to 1.
+This is due to [EIP-161](https://eips.ethereum.org/EIPS/eip-161) which activated in 2016.
+
+This affects the newly deployed dispute games mentioned in ["State Diffs"](#state-diffs).
+
+## State Diffs
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state
+  changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain
+  Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+  ---
+
+### [`0x0d83dab629f0e0f9d36c0cbc89b69a489f0751bd`](https://github.com/ethereum-optimism/superchain-registry/blob/9dc8a7dfb8081291315d0c0ccf871f46c7753b63/superchain/configs/sepolia/unichain.toml#L59)  (OptimismPortal2) - Chain ID: 1301
+
+- **Key:**          `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc`
+
+  - **Decoded Kind:**      `address`
+  - **Before:** `0x2D7e764a0D9919e16983a46595CfA81fc34fa7Cd`
+  - **After:** [`0xB443Da3e07052204A02d630a8933dAc05a0d6fB4`](https://github.com/ethereum-optimism/superchain-registry/blob/9dc8a7dfb8081291315d0c0ccf871f46c7753b63/validation/standard/standard-versions-sepolia.toml#L13)
+  - **Summary:**           ERC-1967 implementation slot
+  - **Detail:**            Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.
```

Comment:
I usually add a little more information here, e.g. OptimismPortal2 contract for `op-contracts/v3.0.0-rc.2`.